lib/stages/CMakeLists.txt

```diff
@@ -48,6 +51,8 @@ add_library(
     bufferDelay.cpp
     BufferSplit.cpp
     InputSubset.cpp
+    mergeBeamBuffer.cpp
+    BeamBufferSort.cpp
```

Comment:
I'm also really bad at this, but we should try to be consistent with naming new stages with a starting capital letter.   (It will make it easier when we go back in and rename all the old lower case stages).

Reply:
So it would be "MergeBeamBuffer.cpp" instead of "mergeBeamBuffer.cpp"?